src/compiler/transformers/module/module.ts

```diff
@@ -1217,15 +1252,7 @@ namespace ts {
         function createAllExportExpressions(name: Identifier, value: Expression, location?: TextRange) {
             const exportedNames = getExports(name);
             if (exportedNames) {
-                // For each additional export of the declaration, apply an export assignment.
-                let expression: Expression = isExportName(name) ? value : createAssignment(name, value);

```

Comment:
`isExportName(name)` didn't seem to have an effect on the output and wasn't used in other places, so I removed it.